pennylane/templates/embeddings.py

```diff
@@ -164,56 +181,48 @@ def BasisEmbedding(features, wires):
     wires, n_wires = _check_wires(wires)
     _check_shape(features, (n_wires,))
 
-    # basis_state cannot be trainable
-    msg = "The input features in BasisEmbedding influence the circuit architecture and can " \
-          "therefore not be passed as a positional argument to the quantum node."
-    _check_no_variable([features], ['features'], msg=msg)
-
```

Comment:
This will also allow BasisEmbedding to receive positional arguments of the qnode.